e2e-tests/monitoring-2-0/conf/monitoring-rs0.yml

```diff
@@ -21,6 +21,33 @@ spec:
           requests:
             storage: 1Gi
     size: 3
+
```

Comment:
can we please add 
```
compare_kubectl statefulset/$cluster-cfg
compare_kubectl deployment/$cluster-mongos
```
lines into `e2e-tests/monitoring-2-0/run` ?
reason: it is needed to see if such changes modify final structure in k8s in fact
please

Reply:
done